geomesa-examples-authorizations/pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <groupId>com.example.geomesa</groupId>
         <artifactId>geomesa-tutorials</artifactId>
-        <version>1.3.3.1-SNAPSHOT</version>
+        <version>1.3.4.0-SNAPSHOT</version>
```

Comment:
Is this for 1.4.x or 1.3.4?

Reply:
i originally did this for 1.3.4 - I've updated for 1.4.0